examples/modals/modal.py

```diff
@@ -0,0 +1,42 @@
+import nextcord
+
+TESTING_GUILD_ID = 123456798  # Replace with your testing guild id
+
+class Pet(nextcord.ui.Modal):
+    def __init__(self):
+        super().__init__(
+            "Your pet",
+            timeout=5*60, # 5 minutes
+        )
+        self.name = nextcord.ui.TextInput(
+            label="Your pet's name",
+            max_length=50,
+        )
+        self.description = nextcord.ui.TextInput(
+            label="Description",
+            style=nextcord.TextInputStyle.paragraph,
+            placeholder="Information that can help us recognise your pet",
+            required=False,
+            max_length=1800,
+        )
+        self.add_item(self.name)
+        self.add_item(self.description)
+    
+    async def callback(self, inter: nextcord.Interaction) -> None:
```

Comment:
Please dont shorten interaction here

Reply:
The code is marked as outdated, I fixed the issue in commits